### PR TITLE
added users tests

### PR DESCRIPTION
**Pull request by sychova**

### CODE REVIEW

File: tests/user.test.js [hunk 1]
```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
...
-    }).expect(201)
+    const response = await request(app)
+        .post('/users')
+        .send({
+            name: 'Anastasiya',
+            email: 'a.sychova@sumatosoft.com',
+            password: 'Qwerty1234'
+        }).expect(201)
 
     // Assert db changed correctly
```
Comment (alexandr-senyuk) [inline tests/user.test.js]: remove redundant comments

File: tests/user.test.js [hunk 2]
```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
...
     const user = await User.findById(response.body.user._id)
     expect(user).not.toBeNull()
 
     // Assertions about the response
-    // expect(response.body.usser.name).toBe('Anastasiya')
```
Comment (alexandr-senyuk) [inline tests/user.test.js:20]: remove

File: tests/user.test.js [hunk 3]
```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
 beforeEach(setupDatabase)
```
Comment (alexandr-senyuk) [inline tests/user.test.js]: you should not setup db each time test file runs, i'd rather use `globalSetup.js` file next time

File: tests/user.test.js [hunk 4]
```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
...
 
 test('Should signup a new user', async() => {
```
Comment (alexandr-senyuk) [inline tests/user.test.js]: same issues as here
https://github.com/sychova/task-manager-api/pull/1

File: tests/user.test.js (hunk 1 above)
Comment (sychova) [reply tests/user.test.js]: Fixed.

File: tests/user.test.js (hunk 2 above)
Comment (sychova) [reply tests/user.test.js:20]: Fixed.

File: tests/user.test.js (hunk 3 above)
Comment (sychova) [reply tests/user.test.js]: Fixed.

File: tests/user.test.js (hunk 4 above)
Comment (sychova) [reply tests/user.test.js]: Could you please clarify which issue exactly do you mean?

Comment (sychova) [reply tests/user.test.js]: Fixed.

**Pull request closed without merging**